ClicksForms.md

```diff
@@ -5,9 +5,15 @@
 
 #### Data we collect 
 > We only store the information about your form once you run the add-on, and is not associated with you or your account
+> 
 > This is stored such that you can enter a code on the bot, and upload the form to ClicksForms
+> 
 > No personal information is stored
+> 
 > No third parties can access this information
+>
+> No third parties will be given access to this information
```

Comment:
Doesn't this repeat the 'no third parties have access to this information' on the above line?

Merge if you like but I feel there may be a little redundancy there